interface/patient_file/deleter.php

```diff
@@ -239,11 +237,13 @@ function popup_close() {
 
                 $res = sqlStatement("SELECT * FROM forms WHERE pid = ?", array($patient));
                 while ($row = sqlFetchArray($res)) {
-                    form_delete($row['formdir'], $row['form_id'], $row['pid'], $row['encounter']);
+                    row_modify(
+                        "forms",
+                        "deleted = 1",
+                        "pid = '" . $row['pid'] . "' AND form_id = '" . $row['form_id'] . "'"
```

Comment:
should use sql escaping here since this goes into a sql query:
```
"pid = '" . add_escape_custom($row['pid']) . "' AND form_id = '" . add_escape_custom($row['form_id']) . "'"
```